docs/source/model_import.rst

```diff
@@ -0,0 +1,40 @@
+Model Import
+============
+
+Snap ML supports importing tree ensembles models that were trained with other frameworks (e.g., scikit-learn, XGBoost, LightGBM) so one can leverage Snap ML's accelerated inference engine.
+One can import a model either by:
```

Comment:
We could add an image here showing the high-level import flow. Without the internal transform details. What do you think?

Reply:
ok, let me know if I should create such a image

Reply:
Yes - please go ahead!